paddle/fluid/platform/dynload/tensorrt.h

```diff
@@ -32,21 +32,24 @@ extern void* tensorrt_dso_handle;
 
 #ifdef PADDLE_USE_DSO
 
-#define DECLARE_DYNAMIC_LOAD_TENSORRT_WRAP(__name)                      \
-  struct DynLoad__##__name {                                            \
-    template <typename... Args>                                         \
-    auto operator()(Args... args) -> DECLARE_TYPE(__name, args...) {    \
-      using tensorrt_func = decltype(&::__name);                        \
-      std::call_once(tensorrt_dso_flag, []() {                          \
-        tensorrt_dso_handle =                                           \
-            paddle::platform::dynload::GetTensorRtDsoHandle();          \
-        PADDLE_ENFORCE(tensorrt_dso_handle, "load tensorrt so failed"); \
-      });                                                               \
-      static void* p_##__name = dlsym(tensorrt_dso_handle, #__name);    \
-      PADDLE_ENFORCE(p_##__name, "load %s failed", #__name);            \
-      return reinterpret_cast<tensorrt_func>(p_##__name)(args...);      \
-    }                                                                   \
-  };                                                                    \
+#define DECLARE_DYNAMIC_LOAD_TENSORRT_WRAP(__name)                         \
+  struct DynLoad__##__name {                                               \
+    template <typename... Args>                                            \
+    auto operator()(Args... args) -> DECLARE_TYPE(__name, args...) {       \
+      using tensorrt_func = decltype(&::__name);                           \
+      std::call_once(tensorrt_dso_flag, []() {                             \
+        tensorrt_dso_handle =                                              \
+            paddle::platform::dynload::GetTensorRtDsoHandle();             \
+        PADDLE_ENFORCE_NOT_NULL(                                           \
+            tensorrt_dso_handle,                                           \
+            platform::errors::Unavaliable("load tensorrt so failed"));     \
```

Comment:
may be here using the so file name is better

Reply:
Done.